app/controllers/products_controller.rb

```diff
@@ -4,10 +4,15 @@ class ProductsController < ApplicationController
   # GET /products or /products.json
   def index
     @products = Product.all.order(:title)
+    respond_to do |format|
+      format.html
+      format.json { render :json => Product.joins(:category).select('products.title,categories.name') }
```

Comment:
new hash syntax
againn spacing issue

Reply:
Further PRs wont be reviewed if I see spacing / indentation issues

Reply:
sorry sir.